.circleci/config.yml

```diff
@@ -158,6 +159,9 @@ jobs:
       - run:
           name: build
           command: make test-binaries -j $JOBS
+      - run:
+          name: install conmon
+          command: cp $CONMONLOCATION bin
```

Comment:
I guess this is needed because the integration tests expect conmon in `$GOPATH/src/github.com/cri-o/cri-o/bin/conmon`. Usually we should not need this because we support this "auto `$PATH` discovery feature". So, simply  removing the conmon path from the integration tests should work, right?

Reply:
I adapted the test_runner to copy conmon from the path into the repo. there's one integration test that needs to modify conmon, so a copy is copied into the $TESTDIR. I'll look into instead modifying that test :+1:

Reply:
Yeah that would be wonderful :+1: Ah I see it's the test which `chmod` to the conmon executable.